src/bot/commands/lastfm/fm.ts

```diff
@@ -153,8 +155,101 @@ export class LastFm extends Command {
                 }
                 break;
             case 'artist':
+                if (input.getArgument('user')) {
+                    let requestURL = request((lastfmURL + 'artist.search' + '&artist=' + queryString), (error: any, response: Response, body: any) => {
+                        if (error) {
+                            input.channel.send(`${Emoji.ERROR}  Connection error! Unable to retrieve lastfm data.`);
```

Comment:
I'm not positive, but I'm fairly certain that unless you explicitly return when the 'error' param is passed, the rest of the function will continue to run regardless. Like I said, not sure, but please look into it.